config/routes.rb

```diff
@@ -15,6 +15,9 @@
       get :refresh_user_list
       post :search
     end
+    member do
+      get :activities
```

Comment:
Add the endpoint /users/<id>/activities to get a list of activities that a user ordered something at.